.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,6 +6,12 @@
   - Fixes (issue number(s))
   - Adds (specific feature) at the request of (project or person)
 
+Note:
+
+***YOUR PR WILL FAIL*** unless you initiate the gitlab CI test suite.
```

Comment:
👍

Reply:
Actually I think tests would be indefinitely "pending". How about:
"Passing gitlab CI test suite is now REQUIRED to get your PR merged."

Reply:
I like that suggestion. Thanks @adrienbernede

Reply:
Yes, I agree. I feel like the way it is now sounds very dooming lol